site/gdocs/components/centered-article.scss

```diff
@@ -807,33 +807,40 @@ p.article-block__text + #article-licence {
     }
 }
 
-.article-block__aside-right figcaption {
-    border-left: 1px solid $blue-20;
-    padding-left: 24px;
-    a {
-        @include owid-link-90;
+figure[class*="article-block__aside"] {
+    position: relative;
+    margin: 0;
+    color: $blue-60;
+    grid-row: span 3 / span 3;
```

Comment:
`grid-row: span 3`? I don't think it's valid to have span for both grid-row-start and grid-row-end

Reply:
Yeah you're right - weird - I thought `span 3` by itself wasn't working right. 🤔